docs/reference/inference/put-inference.asciidoc

```diff
@@ -64,16 +65,47 @@ The type of the {infer} task that the model will perform. Available task types:
 (Required, string)
 The type of service supported for the specified task type.
 Available services:
+* `cohere`: 
 * `elser`: specify the `sparse_embedding` task type to use the ELSER service.
-* `openai`: specify the `text_embedding` task type to use the OpenAI service.
 * `hugging_face`: specify the `text_embedding` task type to use the Hugging Face 
 service.
+* `openai`: specify the `text_embedding` task type to use the OpenAI service.
 
 `service_settings`::
 (Required, object)
 Settings used to install the {infer} model. These settings are specific to the
 `service` you specified.
 +
+.`service_settings` for `cohere`
+[%collapsible%closed]
+=====
+`api_key`:::
+(Required, string)
+A valid API key of your Cohere account. You can find your Cohere API keys or you 
+can create a new one 
+https://dashboard.cohere.com/api-keys[on the API keys settings page].
+
+IMPORTANT: You need to provide the API key only once, during the {infer} model 
+creation. The <<get-inference-api>> does not retrieve your API key. After 
+creating the {infer} model, you cannot change the associated API key. If you 
+want to use a different API key, delete the {infer} model and recreate it with 
+the same name and the updated API key.
+
+`embedding_type`::
+(Required, string)
```

Comment:
Let's switch this to `Optional`. We can say it defaults to `float` if not specified: https://docs.cohere.com/reference/embed under `embedding_types`